examples/asr/conf/citrinet/config_pt.yaml

```diff
@@ -0,0 +1,230 @@
+name: &name "ContextNet5x1"
```

Comment:
If its contextnet, it should be in the contextnet conf folder. But it doesnt seem to be a contextnet below.